rules.ini

```diff
@@ -44,7 +44,16 @@ ChromeEngine[] = \.rpack$
 Clausewitz[] = (?:^|/)common/defines\.(?:lua|txt)$
 Clausewitz[] = ^(?:game/)?common/defines/00_defines\.txt$
 ClickTeamFusion = (?:^|/)mmfs2\.dll$
-Cocos2d = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)Cocos2d\.lua$
```

Comment:
any depot that matches this file have a folder named cocos2d, might be worth it to think about looking for the "string" `cocos2d`, from what i could see there wasn't any false positive

Reply:
Yep, I think we could just use `cocos2d` keyword to match both `folder` and `filename`. I will change them to just `cocos2d`.
```
Cocos[] = cocos2d
```